.node-version

```diff
@@ -1 +1 @@
-16.14
\ No newline at end of file
+18.16
```

Comment:
I've also made this change in our netlify build configs (though I think this is overriding that).